README.md

```diff
@@ -14,70 +14,60 @@ Takeaway Challenge
 ```
+<h3>View the menu</h3>
 
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * Place the order by giving the list of dishes, their quantities and a number that should be the exact total. If the sum is not correct the method should raise an error, otherwise the customer is sent a text saying that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+```rb
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+takeaway.display_menu
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+```
+
+<h3>Select an ice cream</h3>
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+```rb
 
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+takeaway.select_dish("dish")
 
+```
 
-In code review we'll be hoping to see:
+We'll add the dish to your order if it's on the menu.
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+Keep adding items till you're ready to place an order.
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this at this moment.
+<h3>Place an order</h3>
 
-Notes on Test Coverage
-------------------
+```rb
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you run your tests.
+takeaway.place_order("+441234567890")
+
+```
+We'll let you know the bill total and send a confirmation to your phone number
```

Comment:
This instructions were clear and concise. I was able to follow them easily. The only issue is that I received an error when trying to send the confirmation text. 
`Twilio::REST::RestError ([HTTP 404] 20404 : Unable to create record)
The requested resource /2010-04-01/Accounts//Messages.json was not found
https://www.twilio.com/docs/errors/20404`